CONTRIBUTING.md

```diff
@@ -101,6 +101,64 @@ stories/core/**Alert.stories.tsx**
 
 - Contains common demo states and examples for the component.
 
+### Documentation
+
+We use [JSDoc](https://jsdoc.app/) to write documentation for our code.
```

Comment:
I think the main reason we wanted this PR is to prevent confusion with TSDoc, so maybe mention that here.
```suggestion
We use [JSDoc](https://jsdoc.app/) (not TSDoc) to write documentation for our code.
```

Reply:
Good idea.